src/changelog/types.ts

```diff
@@ -0,0 +1,61 @@
+import { Eq } from "fp-ts/lib/Eq";
```

Comment:
the name of this file (`types.ts`) is misleading

Reply:
According on [this](https://github.com/pagopa/danger-plugin/pull/3#discussion_r886422714), we can remove this file!